### ISSUE TITLE
Build python with no dependencies

### ISSUE THREAD

**Issue by seawolf42** [seawolf42/dockerpy#7]
Python builds should have no dependencies on external libraries, the whole thing should be statically linked into a single, self-contained executable.

**Comment by seawolf42** [comment]
According to [Building Python Statically](https://wiki.python.org/moin/BuildStatically) at python.org, this involves compiling as:

    $ ./configure LDFLAGS=”-static” --disable-shared
    $ make LDFLAGS=”-static” LINKFORSHARED=” ”

However, I tried those flags and get:

    configure: WARNING: you should use --build, --host, --target
    configure: WARNING: invalid host type: ./configure
    checking build system type... Invalid configuration `./configure': machine `./configure' not recognized
    configure: error: /bin/bash ./config.sub ./configure failed

... so something is still missing.

We probably need to add a copy of Setup.local (which is basically empty) and copying this into the repo after checking out the version wanted. This should include the keyword `*static*` on a line by itself, at the very least.

**Comment by seawolf42** [comment]
I've started a branch for this. As of 498593f116988fcdc38cb6548055354420339d0a this is not yet working.

With the Setup.local added and building with the flags indicated above, I get the following error:

```bash
checking whether the C compiler works... no
configure: error: in `/root/cpython':
configure: error: C compiler cannot create executables
See `config.log' for more details
```

The referenced `config.log` is attached as `config.log.txt`. Need to analyze.
[config.log.txt](https://github.com/seawolf42/dockerpy/files/1232158/config.log.txt)

**Comment by seawolf42** [comment]
D'oh! Looks like I had curly quotes in the build script (copy/paste without checking). It's building now.

**Comment by seawolf42** [comment]
As of 53f181958441893958e7e9fa3ce2a9d61ec4238c this now appears to build statically, but it's not yet tested in a stand-alone container.

**Comment by seawolf42** [comment]
Fixed with 77b5afa7bed57c0098faa0b1c233f0deb37cd779
